components/countdown-text-area/src/test.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { createMatchers } from 'jest-emotion';
+import * as emotion from 'emotion';
+import { ERROR_COLOUR } from 'govuk-colours';
+import CountdownTextarea from '.';
+
+expect.extend(createMatchers(emotion));
+
+describe('CountdownTextarea', () => {
+  let wrapper;
+
+  describe('displays a countdown based on maxLength prop', () => {
+    it('contains countdown', () => {
+      wrapper = shallow(<CountdownTextarea maxLength={250} />);
+      expect(wrapper.find('StyledCountdown').childAt(0).text()).toBe('250');
```

Comment:
does this really need the `childAt(0)` call?

the previous version didn't have that, and still worked - and is a bit simpler/easier to read

Reply:
With `shallow` it needs this to get to the node value, otherwise `<StyledCountdown />` is returned. With `mount` it works with or without it.

Reply:
ah ok - gotcha :+1:

i faced a similar issue with some code in the sec-env recently - i think i ended up rendering the found child using `.shallow()`.  neither seems like a particularly great thing to have to do, but using either workaround to allow the use of a simple `shallow` seems fair enough

Reply:
Yes, I think it's probably best to avoid using `mount` as much as we can.